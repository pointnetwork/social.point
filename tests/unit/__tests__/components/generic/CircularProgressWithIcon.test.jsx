import React from "react";
import renderer from 'react-test-renderer';

import CircularProgressWithIcon from '../../../../../src/components/generic/CircularProgressWithIcon';

import AddPhotoAlternateOutlinedIcon from '@material-ui/icons/AddPhotoAlternateOutlined';

describe("CircularProgressWithIcon", () => {

    it('should render', () => {
        const component = renderer.create(
            <CircularProgressWithIcon icon={<AddPhotoAlternateOutlinedIcon/>}></CircularProgressWithIcon>
        )
        expect(component.toJSON()).toMatchSnapshot();
    });

});