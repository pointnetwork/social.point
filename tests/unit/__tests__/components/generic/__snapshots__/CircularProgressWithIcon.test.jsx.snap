// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CircularProgressWithIcon should render 1`] = `
<div
  className="MuiBox-root MuiBox-root-1"
>
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
  <div
    className="MuiBox-root MuiBox-root-2"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M18 20H4V6h9V4H4c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2v-9h-2v9zm-7.79-3.17l-1.96-2.36L5.5 18h11l-3.54-4.71zM20 4V1h-2v3h-3c.01.01 0 2 0 2h3v2.99c.01.01 2 0 2 0V6h3V4h-3z"
      />
    </svg>
  </div>
</div>
`;
